src/db/clients/postgresql.js

```diff
@@ -23,31 +23,33 @@ export default function(server, database) {
     const client = new Client(dbConfig);
 
     debug('connecting');
-    client.connect(err => {
+    client.connect(async err => {
       if (err) {
         client.end();
         return reject(err);
       }
 
       debug('connected');
+      const defaultSchema = await getSchema(client);
+
       resolve({
         wrapIdentifier,
         disconnect: () => disconnect(client),
-        listTables: () => listTables(client),
-        listViews: () => listViews(client),
-        listRoutines: () => listRoutines(client),
-        listTableColumns: (db, table) => listTableColumns(client, db, table),
-        listTableTriggers: (table) => listTableTriggers(client, table),
+        listTables: (_, schema = defaultSchema) => listTables(client, schema),
```

Comment:
ok ... then the issue i got is that defaultSchema is taken from the database (get the current schema after login);
as the function definition is outside the default function scope i can't reuse the variable; and need to be connected to determine the schema.
i see 2 other possibilities here:
setting defaultSchema to something static and somewhat standard like 'public
or getting the schema in the body of the function if schema is undefined